src/main/java/ru/yandex/practicum/filmorate/controller/FilmController.java

```diff
@@ -0,0 +1,63 @@
+package ru.yandex.practicum.filmorate.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.*;
+import ru.yandex.practicum.filmorate.exception.FilmException;
+import ru.yandex.practicum.filmorate.exception.ValidationException;
+import ru.yandex.practicum.filmorate.model.Film;
+
+import javax.validation.Valid;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequestMapping(value = "/films", produces = "application/json")
+
+public class FilmController {
+    private final HashMap<Integer, Film> films = new HashMap<>();
```

Comment:
Лучше обозначать интерфейс, а уже инициализировать объект с определенной реализацией. Так программа становится более гибкой и не сломается, если нужно будет подменить реализацию на другую

```suggestion
    private final Map<Integer, Film> films = new HashMap<>();
```